Resources/Maps/edge.yml

```diff
@@ -46349,6 +46360,36 @@ entities:
     - type: Transform
       pos: -39.5,-2.5
       parent: 2
+    - type: EntityStorage
+      air:
```

Comment:
pr

Reply:
I noticed that. Not sure why it saved all that but it's because I told them to open/close the crate to store the 2 batons in it.

Reply:
Maybe just remap the crate and batons. You should be able to store the batons in the crate but... Maybe it's full. So, maybe put those on a rack or something

Reply:
you can remove the air: bit